### ISSUE TITLE
allow setting path.repo, needed for ES snapshots operations

### ISSUE THREAD

**Issue by jjo** [bitnami/bitnami-docker-elasticsearch#73]
**Description**

Make `path.repo` addition to elasticsearch.yml (optionally) configurable

**Steps to reproduce the issue:**

Following https://www.elastic.co/guide/en/elasticsearch/reference/6.7/modules-snapshots.html to create a snapshot with e.g.:
```
PUT /_snapshot/bak

{
  "type": "fs",
  "settings": {
    "location": "/bitnami/elasticsearch/data/snapshot/bak_T35623"
  }
}
```
, fails with:

```
"root_cause": [
  {
    "type": "repository_exception",
    "reason": "[bak] location [/bitnami/elasticsearch/data/snapshot/bak] doesn't match any of the locations specified by path.repo because this setting is empty"
  }
],
```
**Rationale:**

To be able to upgrade between major versions e.g. as described at https://github.com/bitnami/kube-prod-runtime/blob/master/docs/migration-guides/bkpr-1.4-migration-guide.md#backup-kibana-index, (in turn pointing to https://www.elastic.co/guide/en/elasticsearch/reference/6.7/modules-snapshots.html), you need to be able to create (later restore) snapshots.

To do so, [elasticsearch requires `path.repo` setup](https://www.elastic.co/guide/en/elasticsearch/reference/6.7/modules-snapshots.html#_shared_file_system_repository) at it's `elasticsearch.yml` main config file, thus it'd be great to instrument this via an env var like `ELASTICSEARCH_REPO_DIR`, to avoid the need to provide a manually crafted config file.

**Version**

`bitnami/elasticsearch:6.7.2-r0`

**Additional environment details (AWS, VirtualBox, Docker for MAC, physical, etc.):**

This is on kubernetes clusters using BKPR 1.3.x ES+Kibana components, while working on migrating them to latest BKPR release.

**Comment by juan131** [comment]
Thanks @jjo 

I just created a couple of PRs (both container and chart) to include a way to set this configuration setting.

**Comment by jjo** [comment]
> Thanks @jjo
> 
> I just created a couple of PRs (both container and chart) to include a way to set this configuration setting.

Replaced by #75

**Comment by jjo** [comment]
> > Thanks @jjo
> > I just created a couple of PRs (both container and chart) to include a way to set this configuration setting.
> 
> Replaced by #75

closed by mistake, reopened.

**Comment by juan131** [comment]
@jjo the new feature including an environment variable to configure this setting are available on `7.6.2-debian-10-r51` and `6.8.8-debian-10-r48` revisions. Could you give them a try?

You just need to set the environment variable `ELASTICSEARCH_FS_SNAPSHOT_REPO_PATH` to set the `path.repo` property.

**Comment by jjo** [comment]
> @jjo the new feature including an environment variable to configure this setting are available on `7.6.2-debian-10-r51` and `6.8.8-debian-10-r48` revisions. Could you give them a try?

As this is snapshotting 6.x (for 7.x upgrade), I need a 6.x release rather

> 
> You just need to set the environment variable `ELASTICSEARCH_FS_SNAPSHOT_REPO_PATH` to set the `path.repo` property.

ack
